### DISCUSSION TITLE
[SNES] Super Empire Strikes Back CRC32 hash incorrect

### DISCUSSION THREAD

**Discussion by LegendaryLinux** [sanni/cartreader#411] [Need help]
Continued from: https://github.com/sanni/cartreader/issues/409

Super Empire Strikes Back may have an incorrect CRC34 hash in `snes.txt`.  
Can anyone else with this cartridge please dump it and let me know what hash you get?

Here is the log file from my dump:
```

OSCR HW5 V9.0

[+] Super Nintendo

[+] Super Nintendo


Searching database...

Name: SUPEREMPIRESTRI
Type: LoROM FastROM
ICs: ROM ONLY
ROM Size: 16Mbit
Banks: 64 Chips: 0
Sram Size: 0Kbit
ROM Version: 1.0
Checksum: 8FE5
 
 

[+] Read Rom


Saving to SNES/ROM/SUPEREMPIRESTRI/65/...
[*******************]
Internal Checksum...OK
CRC32... 2BF99230 -> Not found
```

**Comment by sanni** [comment]
Should be fixed now. Thanks for reporting the bug. 😃

**Reply by LegendaryLinux** [reply]
I was hoping to validate this information with someone else who has the cart, since it's possible the one I have is a repro. I have included an image of my board and casing
![20220718_205743](https://user-images.githubusercontent.com/6321333/179642601-a0ed1d0c-eab2-4df0-9f77-de845cde6204.jpg)
.

**Reply by LegendaryLinux** [reply]
After some digging, it looks like the board I have is in fact valid based on data from SNES central. Thanks for also looking into this.
http://snescentral.com/pcbboards.php?chip=SHVC-2A0N-01

**Reply by PsyK0p4T** [reply]
According to your pics, your pcb is an original one, and you have the v0 of the game.
So the dump "should" give these results : https://datomatic.no-intro.org/index.php?page=show_record&s=49&n=2882
with CRC32 = f45b15be

Did you try to redump it with the latest version Sanni just pushed ? Let us know the results :)

**Reply by LegendaryLinux** [reply]
Yes, I can confirm the CRC32 is `F45B15BE` using the latest firmware.

**Reply by PsyK0p4T** [reply]
Great ! Everything's solved ;)

**Comment by Chipchip256** [comment]
I have the same problem, the burn in test cartridge does not recognize it, please help me
